README.md

```diff
@@ -70,6 +70,17 @@ us single bullet fire weapons.
 9. Build the VS project
 10. The executables are placed at `../game/hl2.exe` for the client and at `../game/srcds.exe` for the server. Note: this path is outside the repository.
 
+### Building on Linux (To the Make File)
+**DISCLAIMER:** Make sure you have your distro's build essentials, gcc and g++ multilibs installed. This will varry from distro to distro, so I can't include everything here.
+1. Clone `https://github.com/gperftools/gperftools.git`

```

Comment:
This should be `git clone https://github.com/gperftools/gperftools.git -b gperftools-2.0 thirdparty/gperftools-2.0`,
so that the proper version is used and put immediately in the correct dir.

Reply:
You can add `--depth 1` to speed up cloning.

Reply:
Why is this not a submodule?

Reply:
This is really old, and we don't clone dependencies anymore.